definitions/npm/react-dnd_v2.x.x/flow_>=v0.23.x/react-dnd_v2.x.x.js

```diff
@@ -87,14 +89,14 @@ type DragPreviewOptions = {
 }
 
 type ConnectDragSource = (
-  elementOrNode: React$Element<any>
+  elementOrNode: ElementOrNode
   options?: DragSourceOptions
-) => React$Element<any>;
+) => ElementOrNode;
```

Comment:
Yeah, something like this? We can `null` out both HTMLElement and ReactElement this way. Thoughts?
```javascript
type ElementOrNode = React$Element<any> | HTMLElement;
```
```javascript
type ConnectDragSource = <T : ElementOrNode>(
  elementOrNode: T
  options?: DragSourceOptions
) => ?T;
```